taskvine/src/bindings/python3/ndcctools/taskvine/manager.py

```diff
@@ -878,6 +882,23 @@ def submit(self, task):
             self._task_table[task_id] = task
             return task_id
 
+    ##
+    # Check the validity of a FunctionCall task
+    # The name of library should exist, and the function to be executed should exist on that library, too
+    #
+    # @param self    Reference to the current manager object.
+    # @param task    A FunctionCall task
+    def check_funcall_validity(self, task):
+        library_name = task.get_library_name()
+        available_library_names = [library_name for library_name in self._library_table.keys()]
```

Comment:
`available_library_names` is not needed. Do:
```
if library_name not in self._library_table:
   raise ...
```

Reply:
solved, thank you!